presto-raptor/src/test/java/com/facebook/presto/raptor/TestRaptorIntegrationSmokeTest.java

```diff
@@ -142,19 +142,19 @@ public void testCreateBucketedTable()
 
         assertQuery("SELECT * FROM orders_bucketed", "SELECT * FROM orders");
         assertQuery("SELECT count(*) FROM orders_bucketed", "SELECT count(*) FROM orders");
-        assertQuery("SELECT count(DISTINCT \"$shard_uuid\") FROM orders_bucketed", "SELECT 50");
```

Comment:
Number of shards created is dependent on how storage manager config and compaction. Which was causing this test to fail sporadically which is why I took it out.